homeassistant/components/androidtv/media_player.py

```diff
@@ -287,6 +292,33 @@ def service_adb_command(service):
         schema=SERVICE_ADB_COMMAND_SCHEMA,
     )
 
+    def service_learn_sendevent(service):
+        """Translate a key press on a remote to ADB 'sendevent' commands."""
+        entity_id = service.data[ATTR_ENTITY_ID]
+        target_devices = [
+            dev
+            for dev in hass.data[ANDROIDTV_DOMAIN].values()
+            if dev.entity_id in entity_id
+        ]
+
+        for target_device in target_devices:
+            output = target_device.learn_sendevent()
+
+            # log the output, if there is any
+            if output:
+                _LOGGER.info(
```

Comment:
Should we make this a persistent notification instead? That way it's easier to find in the UI.

Reply:
I could do that. I assume this is all I need to do to create a persistent notification:

https://github.com/home-assistant/core/blob/4a65bed0eb9587d0374e03a267a2e630443a4065/homeassistant/components/broadlink/__init__.py#L99-L101